kotlin/src/main/java/app/rive/runtime/kotlin/RiveArtboardRenderer.kt

```diff
@@ -417,7 +419,7 @@ open class RiveArtboardRenderer(
         // https://2dimensions.slack.com/archives/CLLCU09T6/p1638984141105200
         if (settleStateMachineState) {
             activeArtboard?.let {
-                advanceStateMachineInstance(stateMachineInstance, it, 0f)
+                advanceStateMachineInstance(stateMachineInstance, 0f)
```

Comment:
Since now everything's an instance, can't this be `stateMachineInstance.advance()`?